openedx/core/djangoapps/user_api/models.py

```diff
@@ -27,6 +30,10 @@ class UserPreference(models.Model):
     key = models.CharField(max_length=255, db_index=True, validators=[RegexValidator(KEY_REGEX)])
     value = models.TextField()
 
+    TIME_ZONE_CHOICES = [
+        [tz, get_formatted_time_zone(tz)] for tz in common_timezones
```

Comment:
True.  So this code is correct, but I'd still like it changed, because it's not robust.  If we decide to change the way we display timezones, this could easily become unusable without anyone noticing.  Including the sort function makes it more resilient to UX changes.

It won't cause a performance issue, because the sorting will only happen once, when the class is loaded.